src/app/material-docs-app.ts

```diff
@@ -10,7 +10,11 @@ import {CookiePopup} from './shared/cookie-popup/cookie-popup';
 
 @Component({
   selector: 'material-docs-app',
-  templateUrl: './material-docs-app.html',
+  template: `
```

Comment:
I inlined these since it was overkill to have a separate file for 3 lines of HTML.